lib/dolly/document.rb

```diff
@@ -128,7 +128,8 @@ def write_property name, value
     end
 
     def read_property name
-      self.properties[name].value
+      return self.properties[name].value unless doc[name.to_s]
```

Comment:
`doc[name.to_s] || self.properties[name].value`

I think this show a flaw in the property object design. Or on how the default value is set into property.